python/packages/ai/teams/ai/clients/openai/schemas/moderation.py

```diff
@@ -0,0 +1,44 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from dataclasses import dataclass
+from typing import Any, Dict, List
+
+
+@dataclass
+class CreateModerationResponse:
+    @dataclass
+    class Result:
+        flagged: bool
+        categories: Dict[str, bool]
+        category_scores: Dict[str, int]
+
+        @classmethod
+        def from_json(cls, data: Dict[str, Any]) -> "CreateModerationResponse.Result":
```

Comment:
May name the function as `from_dict`? This is how SK names the function, and `from_json` is used to initialize from json string.